Modules/Microsoft365DSC/Examples/Resources/EXOMigrationEndpoint/1-Create.ps1

```diff
@@ -0,0 +1,40 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        EXOMigrationEndpoint "EXOMigrationEndpoint-testIMAP"
+        {
+            AcceptUntrustedCertificates   = $True;
+            Authentication                = "Basic";
+            Credential                    = $Credscredential;
```

Comment:
The example should use ApplicationID,m TenantId, Thumborint. The credential object is not defined here.